contrib/linux/dosbox-staging.metainfo.xml

```diff
@@ -5,31 +5,14 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0-or-later</project_license>
   <name>DOSBox Staging</name>
-  <summary>DOS/x86 emulator focusing on ease of use</summary>
+  <summary>DOS/x86 emulator with advanced features</summary>
   <description>
     <p>
-      DOSBox Staging is a full x86 CPU emulator (independent of host
-      architecture), capable of running DOS programs that require real or
-      protected mode.
-    </p>
-    <p>It features:</p>
-    <ul>
-      <li>A built-in DOS-like console</li>
-      <li>Emulation of several PC variants: (IBM PC, IBM PCjr, Tandy 1000),
-          and CPUs (286, 386, 486, and Pentium I)</li>
-      <li>Graphics chipsets: Hercules, CGA, EGA, VGA, and SVGA</li>
-      <li>Audio solutions: PC Speaker, Tandy Sound System, Disney Sound Source,
-          Sound Blaster series, and Gravis UltraSound</li>
-      <li>CDROM and CD Digital Audio with audio optionally encoded as FLAC,
-          Opus, OGG/Vorbis, MP3 or WAV</li>
-      <li>Joystick emulation working with modern game controllers</li>
-      <li>Serial port emulation including IPX over UDP and Telnet over TCP/IP</li>
-      <li>Hardware-accelerated video output including integer (pixel-perfect)
-          scaling, sharp-bilinear scaling, OpenGL shaders, and more</li>
-    </ul>
-    <p>
-      DOSBox Staging is highly configurable and sufficiently-optimized to run
-      any DOS game on a modern computer.
+      DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices. It emulates an Intel x86 PC running the MS-DOS operating system with the primary aim of running games, but most application programs will work as well.
+
+      It is meant as a (mostly) drop-in replacement for older DOSBox versions. Your existing configurations will continue to work, but you will have access to many advanced features.
+
+      For the full list of supported features, please visit the DOSBox Staging website.
```

Comment:
Don't we need a link to the website here?

Reply:
Thanks for the explanation (below: https://github.com/dosbox-staging/dosbox-staging/pull/2732#issuecomment-1666494042), @rderooy :+1: